test/e2e/util_gatewayapi_test.go

```diff
@@ -676,13 +680,12 @@ func assertHttpRouteConnection(t *testing.T, hostname string, gateway *gatewayap
 	// Obtain the standard formatting of the dnsRecord.
 	dnsRecordName := operatorcontroller.GatewayDNSRecordName(gateway, domain)
 
-	// Make sure the DNSRecord is ready to use.
+	t.Logf("Making sure the DNSRecord %v for domain %q is ready to use...", dnsRecordName, domain)
 	if err := assertDNSRecord(t, dnsRecordName); err != nil {
 		return err
 	}
 
-	// Wait and check that the dns name resolves first. Takes a long time, so
-	// if the hostname is actually an IP address, skip this.
+	t.Logf("Attempting to resolve %s...", hostname)
```

Comment:
To be precise, it won't attempt to resolve the hostname if it is actually an IP address.  This log message is in accurate in that case.

Reply:
I put the comment back in https://github.com/openshift/cluster-ingress-operator/compare/77beec1a8b6af8c8c5ce00f29f7a60683a84cd75..92e89a2a02fccb3d149e3e0491a30f19fe160774.